mypy/pyinfo.py

```diff
@@ -24,7 +24,11 @@ def getprefixes():
 
 def getsitepackages():
     # type: () -> List[str]
-    if hasattr(site, 'getusersitepackages') and hasattr(site, 'getsitepackages'):
+    if (
+        hasattr(site, 'getusersitepackages')
+        and hasattr(site, 'getsitepackages')
+        and site.ENABLE_USER_SITE
```

Comment:
We should still look in the directories returned by `site.getsitepackages` even if `site.ENABLE_USER_SITE` is false unless I'm mistaken.

Reply:
Oh yes. Good catch.